lib/src/common.ts

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+
+import type {
+  SbGatsbyStory,
+  StoryblokBridgeConfigV2
+} from '../types'
+
+
+export const useStoryblokBridge = (
```

Comment:
Do we have an example of how this method is used on Gatsby projects?

I'm asking because there is a convention for React projects where if you name a function `useXXXX`, it somehow means that is a React Hook or contains a React Hook inside, and in this case, I don't see it happening.

For the React SDK we have renamed this function as `registerStoryblokBridge`, adding an alias on the JS SDK:
https://github.com/storyblok/storyblok-js/blob/b89b47d1603952e2affc6ee44759a41fd8f8878f/lib/index.ts#L143

Reply:
@alexjoverm and I agree with you, @fgiuliani ! Created an alias and [followed our React SDK approach](https://github.com/storyblok/gatsby-source-storyblok/pull/363/commits/7b9d4dc6285a96db0efe8b2931c59ffe1f412127).

Reply:
Amazing, thanks for introducing the change @schabibi1! 🙏 

One more small thing, and sorry bothering 😅 as we are introducing the change, should we update the README file?
https://github.com/storyblok/gatsby-source-storyblok#storyblok-bridge

Reply:
@fgiuliani No prob! Yes, it's planned as I created an alias and I'll make a different commit to separate only relevant PR content👍